docs/UserGuide.md

```diff
@@ -133,7 +133,7 @@ Format: `findE KEYWORD [MORE_KEYWORDS]`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
 * Only the name is searched.
 * Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
+* employees matching at least one keyword will be returned (i.e. `OR` search).
```

Comment:
Minor grammatical error, but `employees` should be changed to `Employees` in the next commit.